R/derive_date_vars.R

```diff
@@ -1055,7 +1055,7 @@ convert_date_to_dtm <- function(dt,
   if (lubridate::is.POSIXct(dt)) {
     return(dt)
   } else {
-    if (is_date(dt)) {
+    if (is.instant(dt)) {
```

Comment:
not sure what this update relates to? is it anything to do with this issue?

Reply:
It is not related to this issue but the tests fail. It is related to the update of `is.R` in admiraldev.